src/peergos/server/tests/UserTests.java

```diff
@@ -138,17 +138,17 @@ public void singleSignUp() throws Exception {
 
     @Test
     public void changePassword() throws Exception {
-        String username = "test" + (System.currentTimeMillis() % 10000);
+        String username = "test" + (random.nextInt() % 10000);
```

Comment:
doesn't have to be an int,just wanted it to be random, and deterministic, without high chance of repetition within tests